var/spack/repos/builtin/packages/netlib-lapack/ibm-xl.patch

```diff
@@ -14,17 +14,3 @@
  # HP Fortran
  elseif( CMAKE_Fortran_COMPILER_ID STREQUAL "HP" )
 
---- a/CBLAS/CMakeLists.txt
-+++ b/CBLAS/CMakeLists.txt
-@@ -12,8 +12,8 @@
-                          SYMBOL_NAMESPACE "F77_")
- if(NOT FortranCInterface_GLOBAL_FOUND OR NOT FortranCInterface_MODULE_FOUND)
```

Comment:
btw, any reason to split this part of the patch into a separate `cblas-cmake-fix.patch`?  The stand-alone part looks identical to this one. In other words, if you don't touch patches at all and only change the condition
```
patch('ibm-xl.patch', when='@3.7:')
```
you should get the same end result, right?

Reply:
We wanted to explicitly move out the portion of the patch that could apply in more than just the IBM XL case.

Reply:
ok, if you envision this to be useful later on. Just from this PR alone that is not obvious. Thus my comment.